app/src/main/res/layout/popup_change_username.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    >
+
+    <TextView
+        android:id="@+id/placeholder_popup_change_username"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/text_view_extra_large"
+        android:layout_marginStart="@dimen/hori_small"
+        android:layout_marginTop="@dimen/vert_small"
+        android:layout_marginEnd="@dimen/hori_extra_large"
+        android:gravity="left|fill_vertical"
+        android:hint="@string/enterNewUsername"
+        android:inputType="textCapWords"
+        android:singleLine="true"
+        android:textColor="@color/textPrimary"
+        android:textColorHint="@color/textPrimary" />
+
+    <EditText
+        android:id="@+id/popup_change_username"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/text_view_extra_large"
```

Comment:
I would like to know why a fixed value has been used for TextView and EditText's layout_height? And also why the text size has not been specified for either of them? This would make them use the android default values and hence may lead to undesirable/unexpected behavior.

Reply:
Thanks for reviewing! I have worked on the above suggestions and made a new commit.